back/engines/commercial/id_nemlog_in/app/lib/id_nemlog_in/nemlog_in_verification.rb

```diff
@@ -4,10 +4,22 @@ module IdNemlogIn
   module NemlogInVerification
     include Verification::VerificationMethod
 
+    HOSTS_WITH_AGE_MINIMUM_15 = %w[kobenhavntaler.kk.dk]
+
     def verification_method_type
       :omniauth
     end
 
+    def entitled?(auth)
+      issuer = AppConfiguration.instance.settings['verification']['verification_methods'][0]['issuer']
+      return true unless HOSTS_WITH_AGE_MINIMUM_15.any? { |host| issuer.include?(host) }
+
+      age = auth.extra.raw_info.attributes['https://data.gov.dk/model/core/eid/age'][0].to_i
+      raise Verification::VerificationService::NotEntitledError, 'under_15_years_of_age' if age < 15
+
+      true
+    end
```

Comment:
I assume that both the `issuer` value and `age` values will be available, as I think we want this to fail hard if they are not.

Reply:
n/a after subsequent changes